Gemfile.lock

```diff
@@ -403,7 +412,7 @@ DEPENDENCIES
   activerecord-session_store
   activerecord_where_assoc
   ancestry
-  audited
+  audited!
```

Comment:
why this `!`?

Reply:
Looks like: 
"Dependencies specified with a source other than the main Rubygems index (e.g., git dependencies, path-based, dependencies) have a ! which means they are "pinned" to that source2 (although one must sometimes look in the Gemfile to determine in)."

https://stackoverflow.com/questions/7517524/understanding-the-gemfile-lock-file